lib/api/objectCopy.js

```diff
@@ -65,6 +71,17 @@ function _prepMetadata(sourceObjMD, headers, sourceIsDestination, authInfo,
         utils.getMetaHeaders(sourceObjMD) :
         utils.getMetaHeaders(headers);
 
+    // If tagging directive is REPLACE, but you don't specify the
+    // x-amz-tagging in the request, the destination object will
+    // not have any tags.
```

Comment:
ah, I think the comment would be clearer: 

// If tagging directive is REPLACE but you don't specify any
// tags in the request, the destination object will
// not have any tags.
// If tagging directive is COPY but the source object does not have tags,
// the destination object will not have any tags.